packages/wallet-adapter-core/src/types.ts

```diff
@@ -24,6 +24,20 @@ export interface AptosWalletErrorResult {
   message: string;
 }
 
+type OnNetworkChange = (
```

Comment:
```suggestion
export type OnNetworkChange = (
```

Also, if we make this change, we should probably also update the wallet plugin template with the new type https://github.com/aptos-labs/wallet-adapter-plugin-template/blob/main/src/index.ts#L145